react/README.md

```diff
@@ -104,13 +104,20 @@ _NOTE:_ There is a common practice to use stateless/pure functions over ES6 clas
 
 **[⬆ back to top](#table-of-contents)**
 
+### PureComponent vs. Component
+
+Using `PureComponent` is preferred over `Component` because React provides optimizations in how it checks for changes. Instead of doing a deep comparison of what has changed from the previous state of the component, `PureComponent` implements a `shouldComponentUpdate` that performs simple equality checks for props and state. 
```

Comment:
Can you link to the docs for `PureComponent`? https://facebook.github.io/react/docs/react-api.html#react.purecomponent